packages/video_player/video_player_avfoundation/darwin/video_player_avfoundation/Sources/video_player_avfoundation/FVPVideoPlayerPlugin.m

```diff
@@ -320,6 +335,10 @@ - (instancetype)initWithPlayerItem:(AVPlayerItem *)item
   _videoOutput = [avFactory videoOutputWithPixelBufferAttributes:pixBuffAttributes];
   frameUpdater.videoOutput = _videoOutput;
 
+  _pixelBufferSynchronizationQueue =
```

Comment:
Why is this being stored in two different places, rather than being read out of the frame updater? The two must be the same queue in order for things to work, so storing it twice just introduces a potential source of bugs.